docs/docs/how-tos/graph-api.ipynb

```diff
@@ -1912,23 +1914,10 @@
   },
   {
    "cell_type": "markdown",
-   "id": "903f0da5-8c2c-4a7e-96fb-0b16b4756eff",
-   "metadata": {},
-   "source": [
-    "<div class=\"admonition note\">\n",
-    "    <p class=\"admonition-title\">Note</p>\n",
-    "<p>In the above example, nodes <code>\"b\"</code> and <code>\"c\"</code> are executed concurrently in the same [superstep](../../concepts/low_level/#graphs). What happens in the next step?</p>\n",
-    "    <p>We use <code>add_edge([\"b_2\", \"c\"], \"d\")</code> here to force node <code>\"d\"</code> to only run when both nodes <code>\"b_2\"</code> and <code>\"c\"</code> have finished execution. If we added two separate edges,\n",
-    "    node <code>\"d\"</code> would run twice: after node <code>b2</code> finishes and once again after node <code>c</code> (in whichever order those nodes finish).</p>\n",
-    "</div>"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "id": "c1653341-3215-4ca0-b0e7-9be22f0adaa1",
+   "id": "70e67ced",
    "metadata": {},
    "source": [
-    "</details>"
+    "In the above example, nodes `\"b\"` and `\"c\"` are executed concurrently in the same superstep. We set `defer=True` on node `d` so that it will not execute until all pending tasks are finished. In this case, this means that `\"d\"` waits to execute until the entire `\"b\"` branch is finished."
```

Comment:
```suggestion
    "In the above example, nodes `\"b\"` and `\"c\"` are executed concurrently in the same superstep. We set `defer=True` on node `d` so it will not execute until all pending tasks are finished. In this case, this means that `\"d\"` waits to execute until the entire `\"b\"` branch is finished."
```